learning/intermediate/session-4.js

```diff
@@ -45,8 +45,24 @@ const person8 = null
 // IF they CAN drink then you will console log out there {toastMessage} AND THEN return TRUE
 // IF they can NOT drink, do nothing and return FALSE
 // keep in mind that it's POSSIBLE for the object to be empty or undefined, and therefore the properties {age} and {toastMessage} may NOT be defined
-function canPersonDrink() {
+function canPersonDrink(person) {
   // fill in the function here
+
+  const checkAge = person.age
+  const checkToastMsg = person.toastMessage
+  const validateAge = () => {
```

Comment:
Why do you need this function? Why are you console logging in here? The console logging should be the last thing you worry about. That's not really the point of this exercise. The point is to check the person's age to see if they're 21 or over, then do _something_.